plugins/utils/dashboard.nix

```diff
@@ -14,127 +14,370 @@ in {
-          options = {
-            icon = mkOption {
-              description = "Item icon";
-              type = types.nullOr types.str;
-              default = null;
-            };
+      showPackages =
+        helpers.defaultNullOpts.mkBool
+        false
+        "Whether to hide what packages have looaded. Only available in the 'hyper' theme";
 
-            desc = mkOption {
-              description = "Item description";
-              type = types.str;
-            };
+      project = mkOption {
+        description = "Project showcase options. Only available in the 'hyper' theme";
+        type = with types;
+          nullOr (submodule {
+            options = {
+              enable =
+                helpers.defaultNullOpts.mkNullable
+                types.bool
+                "null"
+                "Whether to enable the showcase of the project list";
+
+              limit =
+                helpers.defaultNullOpts.mkNullable
+                types.int
+                "null"
+                "The limit of projects to be showcased";
 
-            shortcut = mkOption {
-              description = "Item shortcut";
-              type = types.nullOr types.str;
-              default = null;
+              icon =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "The icon of the project section";
+
+              label =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "The label of the project section";
+
+              action =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "The action to be ran when selecting a project, this can be a function";
             };
+          });
+        default = null;
+        example = {
+          enable = true;
+          limit = 10;
+          icon = "»";
+          label = "Find projects' files";
+          action = "Telescope find_files cwd=";
+        };
+      };
+
+      mru = mkOption {
+        description = "Configuration for the 'Most Recently Files' module.";
+        type = with types;
+          nullOr (submodule {
+            options = {
+              limit =
+                helpers.defaultNullOpts.mkNullable
+                types.int
+                "null"
+                "The limit of recently opened files to be shown";
+
+              icon =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "The icon of the module's section";
 
-            action = mkOption {
-              description = "Item action";
-              type = types.nullOr types.str;
-              default = null;
+              label =
+                helpers.defaultNullOpts.mkNullable
+                types.str
+                "null"
+                "The label of the module's section";
+
+              cwd_only =
```

Comment:
cwdOnly. This will require to use an `apply` to re-write the submodule (check other plugins for `apply`)